### PR TITLE
Add examples with special characters for API response filtering

### LINKED ISSUE #2198: Update field & label selectors docs to reflect new patch
### Description
[A PR](https://github.com/sensu/sensu-enterprise-go/pull/865) has been created to fix https://github.com/sensu/sensu-go/issues/3364, and should be released in 5.18.

The following selectors are now working:
- `--label-selector "region == 'us'"` (single or double quotes to identify a string)
- `--label-selector "region == 'us-west-1'"` (dashes and/or colons in string tokens)
- `--field-selector '"entity:whisky" in entity.subscriptions'` (string tokens in set-based operators)

Here are the same selectors in the web UI:

- `labelSelector:region == 'us'`
- `labelSelector:region == 'us-west-2'`
- `fieldSelector:'entity:whisky' in entity.subscriptions`
	- or `fieldSelector:"entity:whisky" in entity.subscriptions`

### Category

This is an issue with:

- [ ] errata (error in doc content)
- [ ] new doc (requires a completely new doc)
- [x] update (add missing or refresh existing content)
- [ ] improvement (improve docs content or format)

### Affected pages
AFAIK only https://docs.sensu.io/sensu-go/latest/api/overview/#response-filtering

e.g.
> The fieldSelector parameter does not work with values that contain special characters like hyphens and underscores.

### Additional context

* Browser(s) and version(s): 
* OS:

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
- Removes warnings that label and field selectors do not work with values with special characters
- Adds examples of using single and double quotes to filter for values with special characters

## Motivation and Context
Closes https://github.com/sensu/sensu-docs/issues/2198

### CODE REVIEW

File: content/sensu-go/5.18/api/overview.md [hunk 1]
```diff
@@ -420,6 +416,20 @@ curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/co
 
 ### Examples
 
+#### Values with special characters
+
+To use a label or field selector with string values that include special characters like hyphens and underscores, place the value in single or double quotes:
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/checks -G \
```
Comment (cwjohnston) [inline content/sensu-go/5.18/api/overview.md]:
```suggestion
curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" -X GET http://127.0.0.1:8080/api/core/v2/entities -G \
```

Comment (cwjohnston) [reply content/sensu-go/5.18/api/overview.md]: * curl shows `HTTP/1.1 405 Method Not Allowed` unless I explicitly specify the `GET` verb and 
* I think `region` is a better fit as an entity label

File: content/sensu-go/5.18/api/overview.md [hunk 2]
```diff
@@ -420,6 +416,20 @@ curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/co
...
+--data-urlencode 'labelSelector=region == "us-west-1"'
+{{< /highlight >}}
+
+{{< highlight shell >}}
+curl -H "Authorization: Bearer $SENSU_ACCESS_TOKEN" http://127.0.0.1:8080/api/core/v2/entities -G \
+--data-urlencode 'fieldSelector='entity:whisky' in entity.subscriptions'
```
Comment (cwjohnston) [inline content/sensu-go/5.18/api/overview.md]:
```suggestion
--data-urlencode 'fieldSelector="entity:i-0c1f8a116b84ea50c" in entity.subscriptions'
```

**Review by cwjohnston — CHANGES_REQUESTED** [review]
Tested these out and found a few changes needed.

File: content/sensu-go/5.18/api/overview.md (hunk 2 above)
Comment (cwjohnston) [reply content/sensu-go/5.18/api/overview.md]: This example should be recognizable as an AWS EC2 instance ID, which might be more relatable to our audience.

Comment (hillaryfraley) [reply content/sensu-go/5.18/api/overview.md]: Thank you for catching that.

**Review by palourde — APPROVED** [review]
👌

**Pull request merged**